components/Tabs/Tabs.js

```diff
@@ -68,4 +79,7 @@ class TabCard {
 - In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
 
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+let tabs = document.querySelectorAll(".tab");
```

Comment:
You can just chain this by doing `let tabs = document.querySelectorAll('.tab').forEach(tab => new TabLink(tab))
`